crypto/err/err.c

```diff
@@ -131,9 +101,7 @@ static ERR_STRING_DATA ERR_str_reasons[] = {
      "called a function you should not call"},
     {ERR_R_PASSED_NULL_PARAMETER, "passed a null parameter"},
     {ERR_R_INTERNAL_ERROR, "internal error"},
-    {ERR_R_DISABLED, "called a function that was disabled at compile-time"},
```

Comment:
We should actually use this more, not take it away...

Reply:
This is a clean-up PR, so leaving something that's never used anywhere in the code seems like a mistake.

Reply:
This is a *function code removal PR*, at least according to its description.

Reply:
I still want you to undo this...